protocols/tests/test_migration/test_migration_helpers.py

```diff
@@ -817,6 +817,26 @@ def test_migrate_questionnaire_rd_300_600(self, fill_nullables=True):
     def test_migrate_questionnaire_rd_300_600_nulls(self):
         self.test_migrate_questionnaire_rd_300_600(fill_nullables=False)
 
+    def test_migrate_questionnaire_rd_600_300(self, fill_nullables=True):
+
+        # tests EQ 300 -> 600
```

Comment:
you mean tests EQ 600 -> 300 right ?

Reply:
yea, I think I might delete all those comments!